recipes-mmx/mmx-cli/mmx-cli.bb

```diff
@@ -0,0 +1,93 @@
+################################################################################
+#
+# mmx-cli.bb
+#
+# Copyright (c) 2013-2021 Inango Systems LTD.
+#
+# Author: Inango Systems LTD. <support@inango-systems.com>
+# Creation Date: 20 Feb 2021
```

Comment:
Can you please tell, if Copyright string (line 5) should mention only year 2021 - per the creation date of this file ?

Reply:
Good question.
I don't know
I've just used the same 1st list of Copyright for all MMX files published in OpenSource
There is no strict rule for that
@vicryab @nmerinov Could you please say you opinion about this question?